lib/whitelistedUrlPrefixes.js

```diff
@@ -27,5 +27,7 @@ module.exports = [
   'https://publishers.basicattentiontoken.org/',
   'https://publishers-staging.basicattentiontoken.org/',
   'https://updates.bravesoftware.com/', // remove this once updates are moved to the prod environment
-  'https://pdfjs.robwu.nl/logpdfjs' // allowed because it gets canceled in tracking protection
+  'https://pdfjs.robwu.nl/logpdfjs', // allowed because it gets canceled in tracking protection
+  'https://publishers-distro.basicattentiontoken.org',
+  'https://publishers-staging-distro.basicattentiontoken.org'
```

Comment:
actually please add a slash to the end of these URLs (otherwise `publishers-distro.basicattentiontoken.org.google.com` etc would match)

Reply:
fixed